ios-base/Extensions/TextFieldExtension.swift

```diff
@@ -9,6 +9,27 @@
 import UIKit
 
 extension UITextField {
+  static func primaryTextField(
```

Comment:
I think we should make this an initializer instead, it sounds a bit repetitive and we currently only have primary so it doesn't mean much.